File: tests/fixtures/java/classes.java
package com.example;

public class Calculator {
    private int value;

    public Calculator() {
        this.value = 0;
    }

    public int add(int a, int b) {
        this.value = a + b;
        return this.value;
    }

    public void print(String message) {
        System.out.println(message);
    }
}

public class Point {
    public double x;
    public double y;

    public Point(double x, double y) {
        this.x = x;
        this.y = y;
    }
}

public interface IProcessor {
    void process(String data);
}

public enum Color {
    RED,
    GREEN,
    BLUE
}

public @interface MyAnnotation {
    String value();
}

File: tests/fixtures/kotlin/classes.kt
package com.example

class Calculator {
    private var value: Int = 0

    fun add(a: Int, b: Int): Int {
        value = a + b
        return value
    }

    fun print(message: String) {
        println(message)
    }
}

data class Point(val x: Double, val y: Double)

interface IProcessor {
    fun process(data: String)
}

enum class Color {
    RED,
    GREEN,
    BLUE
}

object Constants {
    const val MAX_SIZE = 100
}

annotation class MyAnnotation(val value: String)
